Feature/added value results design

bmlancien opened the pull request:
@nesnoj I've tried to style the results content similar to the [calculator](https://sl-app.apps2.rl-institut.de/added_value/).

- I added a tabs component to make it more digestible. Does it make sense? If we keep it, we should probably improve the category names or content. For example "Einnahmen durch Pachterlöse" is not visible for me, only the alert, but it should probably go to another category?
- Not all charts and content were visible for me, so I hope that I didn't miss anything. For example, as mentioned, "Einnahmen durch Pachterlöse", but also some charts in "Gewerbesteuereinnahmen durch Anlagengewinne". That's why it can make the layouts look a bit inconsistent (not always two columns layout)
- I wanted to keep the layout from "Gesamteinnahmen über 25 Jahre" in two columns, but then half of the chart was not visible, so I kept it in one column

Close #186

nesnoj commented:
> @nesnoj I've tried to style the results content similar to the [calculator](https://sl-app.apps2.rl-institut.de/added_value/).

Very cool, looks much better now :+1: 

> * I added a tabs component to make it more digestible. Does it make sense? If we keep it, we should probably improve the category names or content. For example "Einnahmen durch Pachterlöse" is not visible for me, only the alert, but it should probably go to another category?

The tabs are good!
I can see the Pachterlöse row but it was redundant (probably a copy&paste error) looking like this which I fixed now:

![image](https://github.com/user-attachments/assets/f2c1044a-a838-4526-a04f-103c430cff9b)

> * Not all charts and content were visible for me, so I hope that I didn't miss anything. For example, as mentioned, "Einnahmen durch Pachterlöse", but also some charts in "Gewerbesteuereinnahmen durch Anlagengewinne". That's why it can make the layouts look a bit inconsistent (not always two columns layout)

Could you try to pull & rebuild? I fixed an error which may causes this behaviour..
In the section "Gewerbesteuereinnahmen durch Anlagengewinne" the number of charts varies from 1 to 3 depending on what you selected which might be tricky:

![image](https://github.com/user-attachments/assets/bb9aa0dd-0d81-45d2-a017-0c043594b165)

> * I wanted to keep the layout from "Gesamteinnahmen über 25 Jahre" in two columns, but then half of the chart was not visible, so I kept it in one column

That's fine as it is a quite wide chart.

nesnoj commented:
BTW, the charts' layout and colors should be updated but we can do this later..

bmlancien commented:
> Could you try to pull & rebuild? I fixed an error which may causes this behaviour..

I pulled and did `docker-compose -f local.yml up -d --build` but I still cannot see anything in "Einnahmen durch Pachterlöse", just the alert.

> In the section "Gewerbesteuereinnahmen durch Anlagengewinne" the number of charts varies from 1 to 3 depending on what you selected which might be tricky:

Yes, it's a bit tricky, but I think it's okay to leave the charts on top of each others. I fixed the css, it should take half of the width now (but no charts showing for me):

![chrome-capture-2025-4-11](https://github.com/user-attachments/assets/77fd27bb-1488-4f1f-90a6-be59df133e6f)

nesnoj reviewed (approved):
I'll merge this now and move the chart layout problem to #216
